tests/test_collection.py

```diff
@@ -671,3 +671,16 @@ def test_kwargs(self):
 
     def tearDown(self):
         shutil.rmtree(self.tempdir)
+
+
+def test_collection_http():
```

Comment:
@youngpm Could you flag this test with `@pytest.mark.network` so we can skip it with `$ pytest -m "not network"` to make offline development easier?  Same with the test below.